README.md

```diff
@@ -7,62 +7,39 @@ This repository manages the Cincinnati graph for OpenShift.
 
-### Publish a new release
+1. Create a PR.
+1. Merge the PR to master.
+1. Update your local master branch.
+1. Update the labels in quay based on master.
 
-Add JSON for a node under the `nodes` directory.
-The filename and subdirectory structure doesn't have a tooling impact, but the convention is to use `<major>.<minor>/<full-version>.json` to make it easy to get the history for separate z streams (e.g. `git log nodes/4.1`).
-
-Schema:
-
-* `metadata` (optional object)
-    * `url` (optional string): Canonical URI for the release; generally an Errata URI.
-* `payload` (required string): The by-digest pullspec for the release [container image][image].
-* `version` (required string): The name of the release.
-    Currently [a Semantic Versioning 2.0.0][SemVer] string.
-
-For an example, see [`nodes/4.1/4.1.0.json`](nodes/4.1/4.1.0.json).
-
-If you'd rather create the node entry from tooling, you can use:
-
-```console
-$ hack/graph-util.py update-node quay.io/openshift-release-dev/ocp-release:4.1.0
-```
+**Do Not Ever Update Quay Labels Based On Your Local Development Branch. Only from master!**
 
 ### Add a release to a channel
 
-Add a symlink to the `nodes` entry from the `channels/<channel>` directory.
-For example, [`channels/stable-4.1/4.1.0.json`](channels/stable-4.1/4.1.0.json) is a symlink to [`nodes/4.1/4.1.0.json`](nodes/4.1/4.1.0.json), which promotes the 4.1.0 release to the `stable-4.1` channel.
-
-```console
-$ ln -rs nodes/4.1/4.1.0.json channels/stable-4.1/
-```
-
-### Add an upgrade edge between releases
-
-Add JSON for an edge under the `edges` directory.
-The filename and subdirectory structure doesn't have a tooling impact, but the convention is to use `<to-major>.<to-minor>/<to-version>/<from-version>.json` to make it easy to get the history for separate z streams (e.g. `git log edges/4.1`).
+Edit the appropriete file in `channels/`. For example, to add a release to stable-4.2 you would edit `channels/stable-4.2.yaml`
 
-Schema:
+The file contains a list of versions. Please keep the versions in order. And LEAVE COMMENTS if you skip a versions.
 
-* `channels` (required array of strings): Channel names in which the edge exists.
-    This can be a single single channel to allow for per-channel metadata such as phased rollout start times.
-    It can also be multiple channels if you do not need channel-specific metadata.
-* `from` (required string): The release `version` from which the edge originates.
-* `to` (required string): The release `version` where the edge terminates.
+### Block Edges
 
-For an example, see [`edges/4.1/4.1.2/4.1.0.json`](edges/4.1/4.1.2/4.1.0.json).
-
-To create entries for edges baked into a release image's `release-metadata`, use `extract-edges`.
-For example, create edges to 4.1.0 with:
+Create/edit an appropriete file in `blocked_edges/`. For example: to block all incoming edges to a release create a file such as:
```

Comment:
nit: s/appropriete/appropriate/

nit: I like to have each sentence on its own line so I don't have to worry about wrapping and get more useful `git blame` output.  So I'd like a newline before `For example`, but I can use word diffs and such with whatever source style if this is just me being obsessive ;).